### ISSUE TITLE
TypeError with python 3.4 and nbformat 4.0.1

### ISSUE THREAD

**Issue by rehno-lindeque** [kynan/nbstripout#21]
```
$ nbstripout < notebooks/example.ipynb 
Traceback (most recent call last):
  File "/nix/store/n3mk6yx1gcsjxf8y8y3c4pinwdmysqj5-python3.4-nbstripout-0.2.4/bin/.nbstripout-wrapped", line 12, in <module>
    sys.exit(main())
  File "/nix/store/n3mk6yx1gcsjxf8y8y3c4pinwdmysqj5-python3.4-nbstripout-0.2.4/lib/python3.4/site-packages/nbstripout.py", line 184, in main
    write(strip_output(read(sys.stdin, as_version=NO_CONVERT)), sys.stdout)
  File "/nix/store/bimqljy1xyw9j1rfn09dn53avjkzdrzj-python3-3.4.4-env/lib/python3.4/site-packages/nbformat/__init__.py", line 169, in write
    fp.write(s)
  File "/nix/store/bimqljy1xyw9j1rfn09dn53avjkzdrzj-python3-3.4.4-env/lib/python3.4/codecs.py", line 374, in write
    self.stream.write(data)
TypeError: must be str, not bytes
```
*EDIT:* Seems like this might be fixed by https://github.com/kynan/nbstripout/commit/f0056f52a2c007db139d5f234f00d34e98dfd7df?

**Comment by kynan** [comment]
Yes, I'll do a new release. You can of course `pip install git+https://github.com/kynan/nbstripout` at any time.

**Comment by kynan** [comment]
I've just released 0.2.5. Please reopen if the issue persists.

**Comment by rehno-lindeque** [comment]
Thank you, works perfectly
